src/database/session_postgresql.py

```
@@ -20,7 +22,9 @@
     expire_on_commit=False,
 )
 
-sync_database_url = POSTGRESQL_DATABASE_URL.replace("postgresql+asyncpg", "postgresql")
+sync_database_url = POSTGRESQL_DATABASE_URL.replace(
```

Replacing the substring `"postgresql+asyncpg"` with `"postgresql"` to derive the sync URL is pragmatic and works for the current format, but it assumes the exact substring is present. If you ever change the async driver string or the URL structure, this replace may silently fail. Consider building both URLs explicitly or using `urllib.parse` to parse/modify components to be safer and clearer.